src/objective-c/GRPCClient/private/version.h

```diff
@@ -31,7 +31,10 @@
  *
  */
 
-/* This file is autogenerated from template:
- * templates/src/objective-c/GRPCClient/private/version.h.template */
+/* This file is autogenerated from a template file. Please make
+ * modifications to
+ * `templates/src/objective-c/GRPCClient/private/version.h.template`
+ * instead. This file can be regenerated from the template by running
+ * `tools/buildgen/generate_projects.sh`. */
```

Comment:
Another nit: I *think* I remember the style being followed in the ObjC files for block comments is:
```objc
/*
 * blahblah
 */
```